commitlintplugins.test.ts

```diff
@@ -448,10 +448,10 @@ test('header-max-length-with-suggestions5', () => {
 test('header-max-length-with-suggestions6', () => {
     let commitMsgWithThatExceedsHeaderMaxLength =
         "Upgrade foo bla bla bla bla bla bla bla bla bla bla bla bla bla bla";
-    let headerMaxLength1 = runCommitLintOnMsg(commitMsgWithThatExceedsHeaderMaxLength);
+    let headerMaxLength6 = runCommitLintOnMsg(commitMsgWithThatExceedsHeaderMaxLength);
     let expected_message = `"upgrade" -> "update"`
-    expect(headerMaxLength1.status).not.toBe(0);
-    expect((headerMaxLength1.stdout + '').includes(expected_message)).toEqual(true)
+    expect(headerMaxLength6.status).not.toBe(0);
+    expect((headerMaxLength6.stdout + '').includes(expected_message)).toEqual(true)
```

Comment:
@realmarv the commit msg says "add new tests", but you're modifying tests now; sorry to ask this question again, are you using `git gui` to commit? if you were using git gui, you would find this before you commit

Reply:
I use gitk. I guess it's more related to not changing other parts of the code in non-refactoring PR.

Reply:
> it's more related to not changing other parts of the code in non-refactoring PR.

It's BOTH. You should not change other parts of the code in non-refactoring commit, but to catch this mistake early **you need a visual tool to commit your changes**, and that's why I recommend using git gui, to see the diff before you commit it. If gitk doesn't allow you to see the diff at commit time, stop using it now for that.

Reply:
Ok no problem. I was using vscode git to see the diff in commit time